login.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.querySelector(".emailform .innertext"); // 이메일 input
+  const passwordInput = document.querySelector(".pswdform .innertext"); // 비밀번호 input
+  const loginButton = document.querySelector(".login-btn"); // 로그인 버튼
+  const emailLabel = document.querySelector(".emailform"); // 이메일 label
+  const passwordLabel = document.querySelector(".pswdform"); // 비밀번호 label
+  // 이메일 검사
+  emailInput.addEventListener("blur", function () {
+      validateEmail();
+      toggleLoginButton();
+  });
+  // 비밀번호 검사
+  passwordInput.addEventListener("blur", function () {
+      validatePassword();
+      toggleLoginButton();
+  });
+  // 로그인 버튼 클릭 시 /items로 이동
+  loginButton.addEventListener("click", function (event) {
+      if (loginButton.disabled) {
+          event.preventDefault(); // 버튼이 비활성화 상태라면 이동하지 않음
+      } else {
+          window.location.href = "/items";
+      }
+  });
```

Comment:
💊 제안
버튼이 disabled 상태이면 클릭이 안될테니 아래코드면 충분할 것 같습니다.

```suggestion
  loginButton.addEventListener("click", function (event) {
      event.preventDefault(); 
      window.location.href = "/items";
  });
```